.pylintrc

```diff
@@ -414,7 +414,7 @@ max-bool-expr=5
 max-branches=15
 
 # Maximum number of locals for function / method body
-max-locals=20
+max-locals=25
```

Comment:
Is it possible to move some of the logic to a sub-function rather than increasing this limit?

Reply:
Oh I forgot to mention in the PR description, this rule is broken cannot be concatenated with other rule disabling so i had to change this instead.